tools/ci_build/github/azure-pipelines/templates/win-ci-vs-2019.yml

```diff
@@ -253,7 +253,7 @@ jobs:
       - task: PostAnalysis@2
         displayName: 'Guardian Break v2'
         inputs:
-          GdnBreakGdnToolSDLNativeRulesSeverity: Warning
+          GdnBreakGdnToolSDLNativeRulesSeverity: Note
```

Comment:
Very strange that the Guardian tool classified all the warnings as notes(instead of treating them as warnings and errors). This is the main reason why previously this PostAnalysis step didn't help.